### PR TITLE
you can now create lobbies and there is a global state witch holds the user

### PR DESCRIPTION
**Pull request by Arcanedarkness**

### CODE REVIEW

File: frontend/src/components/Lobbies.vue [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <template>
-	<div class="lobby-list">
-		<strong>Lobbies</strong>
-		<ul class="list-group">
-			<li v-for="lobby in lobbies" :key="lobby.id" class="list-group-item">{{ lobby.name }}</li>
-		</ul>
+	<div>
+		<CreateLobbyModal></CreateLobbyModal>
```
Comment (Jensen96) [inline frontend/src/components/Lobbies.vue]: Naming should perhaps be create-lobby-modal for consistency.

File: commonsettings/events.json [hunk 2]
```diff
@@ -2,5 +2,7 @@
 	"CREATE_USER": "create user",
 	"CREATE_LOBBY": "create lobby",
 	"ALL_LOBBIES": "all lobbies",
-	"NEW_LOBBY": "new lobby"
+	"NEW_LOBBY": "new lobby",
+	"JOIN_LOBBY": "user joins a lobby",
```
Comment (FnuGk) [inline commonsettings/events.json]: kan  det ikke bare være `"JOIN_LOBBY": "join_lobby"`

File: frontend/src/main.js [hunk 3]
```diff
@@ -5,11 +5,15 @@ import App from "./App";
...
+import CreateLobbyModal from "@/components/CreateLobbyModal";
+import CreateLobby from "@/components/CreateLobby";
 
 Vue.config.productionTip = false;
 
 Vue.component("create-user", CreateUser);
-Vue.component("frontpage-user", FrontPageUser)
+Vue.component("frontpage-user", FrontPageUser);
+Vue.component("CreateLobbyModal", CreateLobbyModal);
+Vue.component("CreateLobby", CreateLobby);
```
Comment (Jensen96) [inline frontend/src/main.js]: same with naming here

**Review by Jensen96 — COMMENTED** [review]
Read my comments you modafuckar

File: frontend/src/components/CreateLobby.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+	name: "CreateLobby",
+	data() {
+		return {
+			lobbyName: ""
+		};
+	},
+	methods: {
+		onSubmit() {
+			// TODO: Call the api
```
Comment (FnuGk) [inline frontend/src/components/CreateLobby.vue:22]: du har kaldt api?

File: frontend/src/components/CreateUser.vue [hunk 5]
```diff
@@ -5,14 +5,16 @@
 			<input v-model="username" type="text" class="form-control" id="usernameInput" aria-describedby="usernameHelp" placeholder="Enter username">
 			<small id="usernameHelp" class="form-text text-muted">The nickname that other users will see</small>
 		</div>
-		<router-link to="/Dashboard">
-			<button @click="onSubmit" type="submit" class="btn btn-success">Play</button>
-        </router-link>
+		<!-- <router-link to="/Dashboard"> -->
+			<button @click="onSubmit" type="submit" class="btn btn-success" data-dismiss="modal">Play</button>
+        <!-- </router-link> -->
```
Comment (FnuGk) [inline frontend/src/components/CreateUser.vue]: kan vi ikke helt fjerne det udkommenterede kode

**Review by FnuGk — CHANGES_REQUESTED** [review]

**Review by FnuGk — APPROVED** [review]

**Review by Jensen96 — APPROVED** [review]

**Pull request merged**